utility/src/main/java/org/oppia/util/parser/ImageLoader.java

```diff
@@ -0,0 +1,17 @@
+package org.oppia.util.parser;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.request.target.SimpleTarget;
+
+public interface ImageLoader {
```

Comment:
Please add documentation.

Reply:
@veena14cs This comment has not been addressed

Reply:
added doc comment.